### PR TITLE
Updated About Page with Shadow information

### PR DESCRIPTION
**Pull request by apluscs**
Long lines will be formatted after content is approved

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]

File: src/main/webapp/about.html [hunk 1]
```diff
@@ -68,13 +67,21 @@ <h6>As an Interviewer</h6>
       <h6>As an Interviewee</h6>
       <p>
         To search for an interview, navigate to the <a href="search-interview.html">
-        Scheduling Page</a>. You can choose what position you are interested in and see what times 
+        Scheduling Page</a>. Specify that you are searching as an interviewee. You can then choose what position you are interested in and see what times
```
Comment (meagancheese) [inline src/main/webapp/about.html]: Long  line

File: src/main/webapp/about.html [hunk 2]
```diff
@@ -68,13 +67,21 @@ <h6>As an Interviewer</h6>
...
-        Scheduling Page</a>. You can choose what position you are interested in and see what times 
+        Scheduling Page</a>. Specify that you are searching as an interviewee. You can then choose what position you are interested in and see what times 
         and interviewers are available. You will be able to see their company and job, but no more 
         personal information.
         <br>
         Once an interviewer is matched with you, you will get more details in an email. The 
         interviews will be one hour long and hosted on Google Meet.
-      </p> 
+      </p>
+      <h6>As an Shadow</h6>
```
Comment (meagancheese) [inline src/main/webapp/about.html]: *a, not an

File: src/main/webapp/about.html [hunk 3]
```diff
@@ -68,13 +67,21 @@ <h6>As an Interviewer</h6>
...
+      <p>
+        To search for an interview, navigate to the <a href="search-interview.html">
+        Scheduling Page</a>. Specify that you are searching as a shadow. You can then choose what position you are interested in and see what existing interviews you can shadow. After selecting a time, you will be able to see the company and job of the interviewer, but no more personal information.
```
Comment (meagancheese) [inline src/main/webapp/about.html]: Long line. Also, are we planning on sending emails? Let's put a TODO in ScheduledInterviewServlet if so.

**Review by meagancheese — CHANGES_REQUESTED** [review]
Nits and question

File: src/main/webapp/about.html (hunk 3 above)
Comment (apluscs) [reply src/main/webapp/about.html]: l'll make an issue in Git. 
Updated description to say long lines will be formatted after content is approved.

**Review by meagancheese — APPROVED** [review]

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**